setup/ci/wercker.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: single
+title:  "Wercker"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+[Wercker](http://www.wercker.com) is a cloud CI system, which can be 
+enabled within Spinnaker as a pipeline trigger and also as a pipeline stage. The setup required
+is described below.
+
+## Links
```

Comment:
I might suggest putting these links at the bottom, as they are more in-depth reference material.

Reply:
moved